### PR TITLE
[OMP] #6999 Fixing directSalesPrice fixes OpenAccess selection for SubmissionFile

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

**Review by asmecher — CHANGES_REQUESTED** [review]
One item needing clarification, thanks!

File: plugins/importexport/native/filter/NativeXmlChapterFilter.inc.php [hunk 1]
```diff
@@ -148,10 +148,9 @@ public function parseAuthor($n, $chapter)
         $deployment = $this->getDeployment();
 
         $authorId = $deployment->getAuthorDBId($n->getAttribute('author_id'));
-        $primaryContact = $n->getAttribute('primary_contact');
         $seq = $n->getAttribute('seq');
 
-        Repo::author()->addToChapter($authorId, $chapter->getId(), (int) $primaryContact, $seq);
+        Repo::author()->addToChapter($authorId, $chapter->getId(), false, $seq);
```
[the remark this answers is not in the document]
Comment (defstat) [reply plugins/importexport/native/filter/NativeXmlChapterFilter.inc.php:153]: Right now it seems that there is no way to define Primary Contact for a ChapterAuthor within the UI. Also, when creating the Author entity, it seems that this attribute has not been taken into account. There is a primaryContact attribute in both submission_chapters and submission_chapter_authors though. Is there a purpose behind that?

[the remark this answers is not in the document]
Comment (asmecher) [reply plugins/importexport/native/filter/NativeXmlChapterFilter.inc.php:153]: Can you check if that's a regression from the `stable-3_3_0` branch or if it's a long-standing forgotten feature?

[the remark this answers is not in the document]
Comment (defstat) [reply plugins/importexport/native/filter/NativeXmlChapterFilter.inc.php:153]: [This](https://github.com/pkp/omp/blob/3bb7da02701eea030a77e55aea6408f2c9e4ab70/controllers/grid/users/chapter/ChapterGridHandler.inc.php#L309) is a line of code from stable-3_2_1. It seems that OMP never used `primary_contact` column - It uses `seq` column though (both interface and code/db). The user interface of stable_3_3_0 does not allow the user to declare/change the `seq` of the ChapterAuthor anymore.

[the remark this answers is not in the document]
Comment (asmecher) [reply plugins/importexport/native/filter/NativeXmlChapterFilter.inc.php:153]: OK, I'm satisfied that this isn't a regression, thanks!

**Pull request merged**